queries/wing/locals.scm

```diff
@@ -0,0 +1,6 @@
+(block) @local.scope
+(variable_definition_statement
+  name: (identifier) @local.definition
```

Comment:
local queries are also different from upstream (see `CONTRIBUTING.md`; we don't use `@local` scope -- for now).